frontends/web/tsconfig.json

```diff
@@ -1,14 +1,16 @@
 {
     "compilerOptions": {
-        "target": "es5",
+        "target": "es2017",
         "module": "esnext",
         "moduleResolution": "node",
         "sourceMap": true,
         "allowJs": true,
+        "checkJs": false,
         "jsx": "react",
         "jsxFactory": "h",
         "strictNullChecks": true,
-        "experimentalDecorators": true
+        "experimentalDecorators": true,
+        "removeComments": true,
```

Comment:
Did I add too many comments to my TypeScript files? 🙂 You added this just to remove the file size a bit? What is the disadvantage? Are the comments then also missing in the mapped source?

Reply:
yes as we currently have no minifier I expected this to remove the comments, but there still seem to be many comments. I think I just remove this again. No the sourcemaps would still point to the source code with comments (OR ELSE!).